highfidelity/hifi
# Changing defaultscripts to not delete all scripts when deleted

Resolution to: https://highfidelity.fogbugz.com/f/cases/3107/Deleting-defaultScripts-js-while-in-debugging-mode-causes-all-scripts-to-stop

Basically, when deleting defaultscripts in debug mode, it should not delete all other scripts too.

Testing:
 - Load scripts in debug mode
 - Delete defaultscripts
 - Check that all other scripts are retained
